app/lib/features/chat/widgets/bubble_builder.dart

```diff
@@ -48,22 +49,47 @@ class BubbleBuilder extends ConsumerWidget {
             onLeftSwipe: !isAuthor()
                 ? null
                 : () {
-                    ref.read(chatInputProvider.notifier).setReplyWidget(child);
-                    ref.read(chatRoomProvider.notifier).repliedToMessage =
-                        message;
-                    if (!ref.read(chatInputProvider).showReplyView) {
-                      ref.read(chatInputProvider.notifier).toggleReplyView();
+                    if (ref.read(chatInputProvider).emojiRowVisible) {
```

Comment:
this cries for being taken out here into a local function above to be referenced here.... considering that it is being used twice and hard to read in this indentation-level